lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -632,6 +635,17 @@ class MailboxController extends BaseMailboxController with MailboxActionHandlerM
     }
   }
 
+  void _renameMailboxFailure(RenameMailboxFailure failure) {
+    if (currentOverlayContext != null && currentContext != null) {
+      final exception = failure.exception;
+      var messageError = AppLocalizations.of(currentContext!).renameFolderFailure;
+      if (exception is InvalidArgumentsMailboxSetException) {
+        messageError = AppLocalizations.of(currentContext!).folderNameCannotContainSpecialCharacters;
+      }
```

Comment:
I'm not sure `SetError.invalidArguments` only happens for special characters, it might be related to other bugs that we don't know about yet. Please make sure that's always true by official documentation.